--- origsrc/mpgtx-1.3.1/configure	2005-01-02 15:19:23.000000000 -0600
+++ src/mpgtx-1.3.1/configure	2013-11-09 21:26:36.404605800 -0600
@@ -206,7 +206,6 @@ if test $static = "yes"; then
 fi
 
 if test $devel = "no" -a $debug = "no"; then
-lflags="$lflags -s";
 cflags="$cflags $optimization_cflags";
 fi
 
@@ -321,7 +320,7 @@ install:
 	install -d -m 755  \$(INSTALLDIR)
 	install -d -m 755  \$(MANDIR)
 	install -d -m 755  \$(MANDIRDE)
-	install -s -m 755 mpgtx \$(INSTALLDIR)
+	install -m 755 mpgtx \$(INSTALLDIR)
 	cp -\$(cpflags)f mpgdemux mpgjoin mpgcat mpginfo mpgsplit tagmp3 \$(INSTALLDIR)
 	install -m 644 man/mpgtx.1 man/tagmp3.1 \$(MANDIR)
 	install -m 644 man/de/mpgtx.1 man/de/tagmp3.1 \$(MANDIRDE)
